java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -78,4 +79,216 @@ public interface ScriptingAndFunctionsClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> functionLoadWithReplace(String libraryCode, Route route);
+
+    /**
+     * Returns information about the functions and libraries.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above
+     * @see <a href="https://redis.io/docs/latest/commands/function-list/">redis.io</a> for details.
```

Comment:
question: should we be linking redis or valkey right now? or does it not matter, i remember some change of replacing all redis to valkey that was talked about

Reply:
I don't care about links - will update all of them later in bulk.